packages/material-ui/src/InputBase/InputBase.js

```diff
@@ -483,12 +493,7 @@ const InputBase = React.forwardRef(function InputBase(inProps, ref) {
 
   return (
     <React.Fragment>
-      <GlobalStyles
-        styles={{
```

Comment:
Just learned that [`@babel/plugin-transform-constant-elements` deopts mutable props](https://babeljs.io/docs/en/babel-plugin-transform-react-constant-elements). This unlocks a bunch of optimizations actually since we treat props as immutable all the time. Will investigate in a follow-up.